apps/web/app/app.dub.co/(dashboard)/[slug]/(ee)/program/payouts/program-payout-mode-section.tsx

```
@@ -0,0 +1,94 @@
+      label: "Dub and external (Hybrid)",
+      description: (
+        <>
+          Partners that have connected their bank accounts are paid via Dub,
+          others that have tenant ID configured are paid via the{" "}
+          <span className="rounded-md bg-neutral-100 px-1 py-0.5 font-mono">
+            payout.confirmed
+          </span>{" "}
+          webhook event.{" "}
+          <a
+            href="https://dub.co/help/article/partner-payouts#payout-fees-and-timing"
+            target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)